docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -0,0 +1,212 @@
+probes have failed, and the containers have been killed and recreated.
+
+    FirstSeen LastSeen    Count   From            SubobjectPath           Type        Reason      Message
+  --------- --------    -----   ----            -------------           --------    ------      -------
+    37s       37s     1   {default-scheduler }                    Normal      Scheduled   Successfully assigned liveness-exec to worker0
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulling     pulling image "gcr.io/google_containers/busybox"
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulled      Successfully pulled image "gcr.io/google_containers/busybox"
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Created     Created container with docker id 86849c15382e; Security:[seccomp=unconfined]
+    36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Started     Started container with docker id 86849c15382e
+    2s        2s      1   {kubelet worker0}   spec.containers{liveness}   Warning     Unhealthy   Liveness probe failed: cat: can't open '/tmp/healthy': No such file or directory
+
+Wait another minute, and verify that the Container has been restarted:
+
+    kubectl get pod liveness-exec
+
+The output shows that `RESTART` has been incremented:
+
+    NAME            READY     STATUS    RESTARTS   AGE
+    liveness-exec   1/1       Running   1          1m
+
+### Defining a liveness HTTP request
+
+Another kind of liveness probe uses and HTTP GET request. Here is the configuration
+file for a Pld that runs a container based on the `gcr.io/google_containers/liveness`
+image.
+
+{% include code.html language="yaml" file="http-liveness.yaml" ghlink="/docs/tasks/configure-pod-container/http-liveness.yaml" %}
+
+In the configuration file, you can see that the Pod has a single Container.
+The `livenessProbe` field specifies that the kubelet should perform a liveness
+probe every 3 seconds. The `initialDelaySeconds` field tells the kubelet that it
+should wait 3 second before performing the first probe. To perform a probe, the
+kubelet sends an HTTP GET request to the server that is running in the Container
+and listening on port 8080.  If the handler for the server's `/healthz` path
+returns a success code, the kubelet considers the Container to be alive and
+healthy. If the handler returns a failure code, the kubelet kills the Container
+and restarts it.
+
+Any code greater than or equal to 200 and less than 400 indicates success. Any
+other code indicates failure.
+
+You can see the source code for the server in
+[server.go](http://k8s.io/docs/user-guide/liveness/image/server.go).
+
+For the first 10 seconds that the Container is alive, the `/healthz` handler
+returns a status of 200. After that, the handler returns a status of 500.
+
+    http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm used to seeing backticks and a `go` hint around code blocks.  Does it render better as (you may have to look at the soource of this comment):


```go
http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
    duration := time.Now().Sub(started)
    if duration.Seconds() > 10 {
    //...
```